test/GGA.js

```diff
@@ -0,0 +1,33 @@
+const assert = require('assert')
+
+const {createAppWithPlugin} = require ('./testutil')
+
+describe('GGA', function() {
+
+  it('works with default values', done => {
+     const onEmit = (event, value) => {
+        assert.equal(value ,'$IIGGA,172814,3723.4659,N,12202.2696,W,0,0,0,0,M,0,M,,*52')      
+        done()
+     }
+     const app = createAppWithPlugin(onEmit, 'GGA')
+     app.streambundle.getSelfStream('navigation.datetime').push('2015-12-05T17:28:14Z')
+     app.streambundle.getSelfStream('navigation.position').push({ longitude: -122.03782631066667, latitude: 37.39109795066667 })
+  })
+
+/*  it('works with sample values', done=> {
```

Comment:
What about this commented test?

Reply:
I don't know what to do. I am at the end with my Latin. I don't succeed in feeding the test with values. That's why I commented this part out. 
```
      AssertionError [ERR_ASSERTION]: '$IIGGA,172814,3723.4659,N,12202.2696,W,0,0,0,0,M,0,M,,*52' == '$IIGGA,172814.0,3723.46587704,N,12202.26957864,W,2,6,1.2,18.893,M,-25.669,M,2.0,0031*58'
      + expected - actual

      -$IIGGA,172814,3723.4659,N,12202.2696,W,0,0,0,0,M,0,M,,*52
      +$IIGGA,172814.0,3723.46587704,N,12202.26957864,W,2,6,1.2,18.893,M,-25.669,M,2.0,0031*58
      
      at Object.onEmit [as emit] (test/GGA.js:19:16)
      at plugin.unsubscribes.push.Bacon.combineWith.filter.changes.debounceImmediate.onValue.nmeaString (index.js:54:17)
      at node_modules/baconjs/dist/Bacon.js:1048:16
      at node_modules/baconjs/dist/Bacon.js:463:23
      at processAfters (node_modules/baconjs/dist/Bacon.js:359:11)
      at Object.inTransaction (node_modules/baconjs/dist/Bacon.js:439:9)
      at Dispatcher.push (node_modules/baconjs/dist/Bacon.js:1214:24)
      at Dispatcher.handleEvent (node_modules/baconjs/dist/Bacon.js:1267:17)
      at Bus.sink (node_modules/baconjs/dist/Bacon.js:242:17)
      at Bus.push (node_modules/baconjs/dist/Bacon.js:2371:21)
      at Context.done (test/GGA.js:24:60)
```

Reply:
Push position last, after all the other values.

Reply:
thx. that's it!!! 
Another small question: formating of output 
For Example: If I define
app.streambundle.getSelfStream('navigation.gnss.differentialAge').push(2.0)
Result ist integer 2.
How to keep float 2.0?